docs/howto/operator.rst

```diff
@@ -2327,3 +2327,403 @@ More information
 
 See `Google Cloud Vision Product delete documentation
 <https://googleapis.github.io/google-cloud-python/latest/vision/gapic/v1/api.html#google.cloud.vision_v1.ProductSearchClient.delete_product>`_.
+
+Google Cloud Transfer Service Operators
```

Comment:
Recently we have discussed this topic in the team. This link is also missing in other places. We wanted to introduce it in a separate change. For now, we prefer to keep style with other operators.

Reply:
I created a PR that split documentation for operators to multiple file: https://github.com/apache/airflow/pull/4814
When the change is accepted, it will expand the descriptions for each set of operators to explain what service the operators are integrating with. Currently, there is no place to put such information.